app/views/vm_common/_policies.html.haml

```diff
@@ -5,28 +5,9 @@
   %h3
     = _('Policy Simulation Details')
 
-  - squash_state = session[:squash_open] ? "up" : "down"
-  - squash_title = session[:squash_open] ? _("Collapse All") : _("Expand All")
-  %button.btn.btn-default{:title => squash_title,
-                          :remote => true,
-                          :id => 'squash_img',
-                          "data-method" => :post,
-                          "data-miq_sparkle_on" => true,
-                          "data-click_url" => {:url => url_for(:action => 'squash_toggle', :id => @record.id)}.to_json}
-    %i{:class => "fa fa-angle-double-#{squash_state} fa-lg"}
+  .btn.btn-default{:id => "squash_button", :onclick => "miqSquashToggle('#{j_str(@policy_simulation_tree.name.to_s)}')", :title => _('Expand All')}
+    %i{:class => "fa fa-plus-square-o fa-lg"}
 
+  = render(:partial => 'shared/tree', :locals => {:tree => @policy_simulation_tree, :name => @policy_simulation_tree.name})
 
-  %div{:id => "#{session[:tree_name]}box"}
-
-  = render(:partial => "layouts/dynatree",
-    :locals => {:tree_id           => "#{session[:tree_name]}box",
-      :tree_name                   => session[:tree_name],
-      :json_tree                   => session[:policy_tree],
-      :icon_size                   => settings(:views, :treesize).to_i,
-      :base_id                     => "h_#{@record.name}",
-      :cfme_no_click               => true,
-      :tree_state                  => true,
-      :cookie_id_prefix            => "edit_treeOpenStatex",
-      :open_close_all_on_dbl_click => true})
-
-  = _('* Items in <font color="red"><i>red italics</i></font> do not change the outcome of the scope or expression.').html_safe
+  = _('* Items in <font color="red">red</font> do not change the outcome of the scope or expression.').html_safe
```

Comment:
@dclarizio I see and reverted the change. I checked code and there should be a case when there are red italics (but didn't find an example). Exactly same line is in rsop_tree but at top of it and here it's at bottom. Maybe one of them should be moved to match the other?
rsop_tree
![before](https://cloud.githubusercontent.com/assets/9210860/16043205/e0b19ce6-323e-11e6-8c98-1baa14047b8b.png)